src/templates/XmlRenderer/template.tsx

```diff
@@ -0,0 +1,967 @@
+                              <div className="divTableCell">
+                                <div className="divTable">
+                                  <div className="divTableBody">
+                                    <div className="divTableRow">
+                                      <div className="divTableHead">PriceAmount</div>
+                                      <div className="divTableCell">
+                                        <div className="divTable">
+                                          <div className="divTableBody">
+                                            <div className="divTableRow">
+                                              <div className="divTableHead">#text</div>
+                                              <div className="divTableCell">{data.Price.PriceAmount}</div>
+                                            </div>
+                                            <div className="divTableRow">
+                                              <div className="divTableHead">@_currencyID</div>
+                                              <div className="divTableCell" />
+                                            </div>
+                                          </div>
+                                        </div>
+                                      </div>
+                                    </div>
+                                    <div className="divTableRow">
+                                      <div className="divTableHead">BaseQuantity</div>
+                                      <div className="divTableCell">
+                                        <div className="divTable">
+                                          <div className="divTableBody">
+                                            <div className="divTableRow">
+                                              <div className="divTableHead">#text</div>
+                                              <div className="divTableCell">{data.Price.BaseQuantity}</div>
+                                            </div>
+                                            <div className="divTableRow">
+                                              <div className="divTableHead">@_unitCode</div>
+                                              <div className="divTableCell" />
+                                            </div>
+                                          </div>
+                                        </div>
+                                      </div>
+                                    </div>
+                                  </div>
+                                </div>
+                              </div>
+                            </div>
+                          ))}
+                        </div>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </TemplateContainer>
+  );
```

Comment:
oh man, could have divided this into smaller chunks and composed them together later
im wondering how would anyone maintain this template in the future, the person has to count divs to understand where he is at.

Reply:
I copied from the HTML file they provided. I guess will revamp the whole styling and structure.

Reply:
As discussed with @cavacado, we believe that there isn't a need to refactor.